package.json

```diff
@@ -102,6 +122,8 @@
     "prettier": "^2.0.0",
     "prettier-plugin-packagejson": "^2.0.0",
     "prop-types": "^15.7.0",
+    "react-dnd": "^11.1.3",
```

Comment:
```suggestion
    "react-dnd": "^11.0.0",
```

Reply:
We probably don't need this peer dependency at this point since this is only a runtime dependency for the editor itself.

Reply:
removed